Python/ceval.c

```diff
@@ -6432,6 +6432,10 @@ _PyEvalFramePushAndInit(PyThreadState *tstate, PyFunctionObject *func,
 static void
 _PyEvalFrameClearAndPop(PyThreadState *tstate, _PyInterpreterFrame * frame)
 {
+    PyObject **base = (PyObject **)frame;
```

Comment:
Is possible that the compiler will complain that in non debug mode base is unused.

Reply:
Oh yeah. I'll just inline the cast in the assert.